packages/backend-for-frontend/src/services/notificationConfigService.ts

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable max-params */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { bffApi, notificationConfigApi } from "pagopa-interop-api-clients";
+import { WithLogger, assertFeatureFlagEnabled } from "pagopa-interop-commons";
+import { NotificationConfigProcessClient } from "../clients/clientsProvider.js";
+import { BffAppContext } from "../utilities/context.js";
+import { config } from "../config/config.js";
+import {
+  toBffApiTenantNotificationConfig,
+  toBffApiUserNotificationConfig,
+} from "../api/notificationConfigApiConverter.js";
+
+export function notificationConfigServiceBuilder(
+  notificationConfigClient: NotificationConfigProcessClient
+) {
+  return {
+    getTenantNotificationConfig: async ({
+      authData: { organizationId },
+      logger,
+      headers,
+    }: WithLogger<BffAppContext>): Promise<bffApi.TenantNotificationConfig> => {
+      assertFeatureFlagEnabled(config, "featureFlagNotificationConfig");
```

Comment:
We discussed with @sandrotaje and thought it best to check the feature flag here, and not just in the notification-config-process, so that, in environments where the notification-config-process isn't deployed at all for now, the request would be rejected without trying to call the notification-config-process, rather than timing out